Automattic/wp-calypso
# Marketing/Traffic: get rid of `SeoSettingsMain`

Follow up to #59107

#### Changes proposed in this Pull Request

* Remove `SeoSettingsMain` component

The `<SeoSettingsMain />`  component can be removed, because it's only async loading the `SeoSettingsForm` component which can be done in the parent as well. The usage of `<QuerySitePurchases />` seems to be unnecessary as none of the children of the form component make any use of the data (it's necessity isn't clear for me even from looking at the history of the file; it's been introduced with the creation of the file quite a while ago with https://github.com/Automattic/wp-calypso/pull/7688).

#### Testing instructions

* Go to `/marketing/traffic/:site`
* Make sure everything renders and behaves as on prod
